src/main/java/ui/Ui.java

```diff
@@ -68,7 +68,7 @@ public void showRevisionContent(ArrayList<Card> cards, int cardCount, Chapter to
         out.println("The revision for " + toRevise + " will start now:");
         boolean isEqualsDate = false;
         for (Card c : cards) {
-            if (c.getDate().equals(LocalDate.now())) {
+            if (Scheduler.isDeadlineDue(c.getDueBy())) {
```

Comment:
@neojiaern simplified comparing of date to: if dueBy is before or equals to todays date, can be used to reference for revision in card version.